rskj-core/src/main/java/co/rsk/peg/federation/FederationSupportImpl.java

```diff
@@ -667,6 +681,87 @@ protected Integer commitFederation(boolean dryRun, Keccak256 hash, BridgeEventLo
         return FederationChangeResponseCode.SUCCESSFUL.getCode();
     }
 
+    /**
+     * Commits the currently pending federation to be the proposed one.
+     * That is, the proposed federation is set to be the currently pending federation,
+     * and the pending federation is wiped out.
+     * @param dryRun whether to just do a dry run
+     * @param hash the pending federation's hash. This is checked the execution block's pending federation hash for equality.
```

Comment:
`This is checked the execution block's pending federation hash for equality.`. I don't get it is easy to get this. Could you reword it?